hugegraph-llm/src/hugegraph_llm/demo/rag_web_demo.py

```diff
@@ -99,12 +90,13 @@ def build_kg(file, schema, example_prompt, build_mode):  # pylint: disable=too-m
             text += para.text
             text += "\n"
     elif full_path.endswith(".pdf"):
+        # TODO: support PDF file
         raise gr.Error("PDF will be supported later! Try to upload text/docx now")
     else:
         raise gr.Error("Please input txt or docx file.")
     builder = KgBuilder(LLMs().get_llm(), Embeddings().get_embedding(), get_hg_client())
 
-    if build_mode != "Rebuild vertex index":
+    if build_mode != BuildMode.REBUILD_VERTEX_INDEX:
```

Comment:
Maybe this should be `build_mode != BuildMode.REBUILD_VERTEX_INDEX.value`?

Reply:
right